selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/PullRequestPanel.java

```diff
@@ -195,6 +199,11 @@ public void selectBranch(String branchName) {
         By.xpath(String.format(PullRequestLocators.BRANCH_NAME, branchName)));
   }
 
+  /** Click 'refresh contribution branch name' button on the 'Pull Request' panel */
+  public void clickRefreshContributionBranchButton() {
+    seleniumWebDriverHelper.waitAndClick(refreshBranchListButton);
```

Comment:
`refreshBranchListButton` variable name hasn't completely agreed with button name expressed in method name `refreshContributionBranchButton`